hazelcast/src/main/java/com/hazelcast/cp/IAtomicReference.java

```diff
@@ -27,11 +27,11 @@
  * to the {@link java.util.concurrent.atomic.AtomicReference}.
  * <p>
  * Asynchronous variants have been introduced in version 3.7.
- * Async methods immediately return an {@link ICompletableFuture} from which
+ * Async methods immediately return an {@link CompletionStage} from which
  * the operation's result can be obtained either in a blocking manner or by
  * registering a callback to be executed upon completion. For example:
  * <pre>
- * ICompletableFuture&lt;E&gt; future = atomicRef.getAsync();
+ * CompletionStage&lt;E&gt; future = atomicRef.getAsync();
```

Comment:
The following example must be also updated. The replacement for `ICompletableFuture.andThen` is `CompletionStage.whenCompleteAsync`, eg see https://github.com/hazelcast/hazelcast/blob/83d814eefb9864187b3a43736d29422a1581f0aa/hazelcast/src/main/java/com/hazelcast/map/IMap.java#L650-L657